docs/single-spa-api.md

```diff
@@ -1,19 +1,13 @@
 # single-spa API
 The single-spa library does not `export default`, but instead exports named functions and variables.
 
-## setLoader
-`setLoader(Loader)` sets the javascript [loader](https://whatwg.github.io/loader/) that will be used by single-spa.
-A loader must implement `Loader.import(...).then(...).catch(...)`, and the most commonly used loader is
-[SystemJS](https://github.com/systemjs/systemjs). This API should be called **before** any `declareChildApplication`
-calls are made.
-
 ## start
 `start()` is a function that must be called by your root application. Before `start` is called, child
 applications will be loaded, but will never be bootstrapped, mounted or unmounted.
 
 ## declareChildApplication
-`declareChildApplication(name, activeWhen)` is the most important api and the only api that is required to be
-used in order for single-spa to work. It is described in detail inside of the [root-application.md docs](/docs/root-application.md#declaring-child-applications)
+`declareChildApplication(name, activeWhen)` is the most important api your root application will use.
+It is described in detail inside of the [root-application.md docs](/docs/root-application.md#declaring-child-applications)
 
 ## triggerAppChange
 `triggerAppChange()` takes in no arguments and returns a Promise that will resolve/reject when all apps that
```

Comment:
Maybe a slight rephrase of `takes in no arguments and returns a Promise that will resolve/reject when all apps that should be mounted are mounted, etc` to: `takes in no arguments and returns a Promise that will resolve/reject when all apps have successfully mounted`.